src/components/filters/FilterDropdown.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+import React, { useEffect, useState } from 'react';
+import DropdownButton from '@/components/ui/dropdown';
+import { PrismaClient } from '@prisma/client';
```

Comment:
Please, use instead, this is the client as a singleton for a proper use on Next.js
```suggestion
import prisma from "@/lib/db";
```